### PR TITLE
Repo Clean Up Installment 2: S3/ Models/ Data Transfer

### PR DESCRIPTION
**Pull request by kroop-olivia-bah**
# Description

This pull request is part of the effort to clean up the ML repo. It focuses on areas related to S3 models and data transfer operations. It builds off of the changes in the [PR for installment 1](https://github.com/dod-advana/gamechanger-ml/pull/140).

**Ticket(s) or Issue(s):** None

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Local
- [x] Server 119
- [ ] Server 115
- [ ] K8

**Screenshot of automated jenkin tests (if applicable)**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have used an autoformatter like lint or blackc

- [x] This requires a code review

### CODE REVIEW

**Comment by kroop-olivia-bah** [comment]
Screen Shot 2022-07-20 at 12 10 41 PM

**Comment by cWren0110** [comment]
I am okay with these changes, but @rha930 @JaredJRoss it would be good to get your eyes on it to make sure nothing major is impacted.

**Comment by kroop-olivia-bah** [comment]
@cWren0110 I built this branch on 119 and tested the app on 112. No issues encountered

**Comment by rha930** [comment]
Hi, this is going to a require an in depth review and test. The automatic checks wont cover this since it has to do with s3. Can you reopen this PR into dev. 

Thanks

File: gamechangerml/src/data_transfer/delete_local_corpus.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    from gamechangerml.api.utils.
+
+    Args:
+        corpus_dir (str, optional): Directory containing corpus files. Defaults 
+            to "corpus".
+        logger (logging.Logger or None, optional): If None, uses 
+            configure_logger(). Default is None.
+    
+    Returns:
+        int: 0 if success, 1 if exception.
```
Comment (JaredJRoss) [inline gamechangerml/src/data_transfer/delete_local_corpus.py:19]: in s3_download.py the check for delete_local_corpus is if not success which if 0 is returned for success returns true and exits early from the file running

Comment (kroop-olivia-bah) [reply gamechangerml/src/data_transfer/delete_local_corpus.py]: Thanks! Changes pushed

**Comment by JaredJRoss** [comment]
Everything looks good to me

**Review by JaredJRoss — APPROVED** [review]

**Review by rha930 — APPROVED** [review]

**Pull request merged**